bxin/ptg2m1m3/m1m3_diagnostic.ipynb

```diff
@@ -6,8 +6,11 @@
    "source": [
```

Comment:
<p>Defining the log message with the test case number and a descriptive name for the test would be good. </p><p>Also, I think it would be good to send a log message to EFD with the log message, a description of the test and the timestamp. This will make it easier later to go back.</p><p>Not sure, if this comes later in this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>